tasks/backup_transfer.sh

```diff
@@ -18,3 +18,4 @@ if [ "$MANUAL_MIGRATION" = true ]
 fi
 
 rsync -ave 'ssh -o StrictHostKeyChecking=no' "$PT_backupdir"/* root@"$PT_targethost":"$PT_targetdir"
+
```

Comment:
What does the copy on line 16 do?  Why is it copying from the code directory?  Bolt copies everything in the module's `files` directory to the target, so not sure about this.